packages/front-end/components/Settings/EditDataSourceSettingsForm.tsx

```diff
@@ -255,14 +255,18 @@ const EditDataSourceSettingsForm: FC<{
                       <div className="row">
                         <div className="col">
                           <Field
-                            label="SQL Query"
-                            textarea
-                            minRows={10}
-                            maxRows={20}
+                            label="SQL"
```

Comment:
I'm a little confused here how I'm not explicitly setting the `placeholder` but it's somehow working correctly. Need to dive into this a bit further.

Reply:
Er - just realized I was looking at currentValue, not the placeholder. Will need to go back and add placeholder.